docs/src/api/params.md

```diff
@@ -247,6 +247,37 @@ The file path to save the storage state to. If [`option: path`] is a relative pa
 current working directory. If no path is provided, storage
 state is still returned, but won't be saved to the disk.
 
+## js-python-context-option-har
+* langs: js, python
+- `har` <[Object]>
+  - `path` <[path]> Path to a [HAR](http://www.softwareishard.com/blog/har-12-spec) file with prerecorded network data. If the HAR file contains an entry with the matching URL and HTTP method, then the entry's headers, status and body will be used to fulfill the network request. An entry resulting in a redirect will be followed automatically. If there is no matching entry in the file the execution continues to try other configured HAR files and [Route] handlers. If `path` is a relative path, then it is resolved relative to the current working directory.
```

Comment:
`If there is no matching entry in the file the execution continues to try other configured HAR files and [Route] handlers` is not true anymore.